### PR TITLE
[Gpr_To_Absl_Logging] Reduce log noise

### PR DESCRIPTION
**Pull request by tanvi-jagtap**
[Gpr_To_Absl_Logging] Reduce log noise

### CODE REVIEW

File: src/core/ext/transport/chaotic_good/server/chaotic_good_server.cc [hunk 1]
```diff
@@ -510,7 +510,8 @@ int grpc_server_add_chaotic_good_port(grpc_server* server, const char* addr) {
   } else if (!error_list.empty()) {
     LOG(INFO) << "Failed to bind some addresses for " << addr;
     for (const auto& error : error_list) {
-      LOG(INFO) << "  " << error.first << ": " << error.second;
+      GRPC_TRACE_LOG(chaotic_good, INFO)
+          << "Binding Failed: " << error.first << ": " << error.second;
```
Comment (sastryvp) [inline src/core/ext/transport/chaotic_good/server/chaotic_good_server.cc:514]: "Binding failed" isn't there in the original message

File: src/core/handshaker/http_connect/http_connect_handshaker.cc [hunk 2]
```diff
@@ -296,8 +296,8 @@ void HttpConnectHandshaker::DoHandshake(
   // Log connection via proxy.
   std::string proxy_name(grpc_endpoint_get_peer(args->endpoint.get()));
   std::string server_name_string(*server_name);
-  LOG(INFO) << "Connecting to server " << server_name_string
-            << " via HTTP proxy " << proxy_name;
+  VLOG(2) << "Connecting to server " << server_name_string << " via HTTP proxy "
```
Comment (sastryvp) [inline src/core/handshaker/http_connect/http_connect_handshaker.cc:299]: Changed to VLOG to reduce log noise?

**Review by sastryvp — APPROVED** [review]

File: src/core/handshaker/http_connect/http_connect_handshaker.cc (hunk 2 above)
Comment (tanvi-jagtap) [reply src/core/handshaker/http_connect/http_connect_handshaker.cc:299]: Yes.

File: src/core/ext/transport/chaotic_good/server/chaotic_good_server.cc (hunk 1 above)
Comment (tanvi-jagtap) [reply src/core/ext/transport/chaotic_good/server/chaotic_good_server.cc:514]: Wrote it to improve readability of the comment.

**Pull request closed without merging**